app/splits/[id]/split-game-match-details-panel.tsx

```diff
@@ -0,0 +1,35 @@
+import ChampCard from "@/app/components/champ-card";
+import ThreeStateSelection from "@/app/components/three-state-selection";
+import { Box, Button, Text, VStack } from "@chakra-ui/react";
+
+function SplitGameMatchDetailsPanel({ splitGame, onUpdate }: { splitGame: SplitGame, onUpdate: (splitGame: SplitGame) => void }) {
+  function onWinChanged(win: boolean | null) {
+    onUpdate({
+      ...splitGame,
+      win: win
```

Comment:
You should be able to write it like this if var and the param are the same name
```suggestion
      win
```